### PR TITLE
Add readme - initial version

### PR DESCRIPTION
**Pull request by velychkovskyi**
## What Are We Doing Here?

Added Readme file with application description

## How to Test and Verify

Documentation

### Level

`Low`

### Required Testing

`null`

### Risk Summary

No risk

## Code Review Checklist

- [ ] Testing instructions are included. If not, make sure you leave us a line or two for the reason.

- [ ] Well documented, including updates to any necessary markdown files. When
we inevitably come back to this code it will only take hours to figure out, not
days.

- [ ] Make sure PR has only the changes required for the Jira issues.

- [ ] Have you make sure to follow the agreements in our process document?

- [ ] Have someone from different region than you review your changes. If not, make sure to leave a comment on why it is not required

- [ ] If it's applicable, document deployment plan for the changes.

### CODE REVIEW

**Review by vecinomio — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+##### Jenkinsfile 
+Is Managing deployment of new AWS CFN Stack. Using "Build with parameters" we can manipulate with new AWS CloudFormation stack parameters.
+
+##### Jenkinsfile2 
+Is Managing Application Load Balancer parameters and can destroy AWS CFN stack with outdated version of application. 
+
+Link to Jenkins:
+https://ci.devopsa3.me.uk/
+
+Link to origin application:
```
Comment (Emperorex) [inline README.md]: I suggest to re-phrase it like:

> Binaries location

**Review by Emperorex — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (velychkovskyi) [reply README.md]: Actually, it is not binaries, so I changed it to our application link

**Review by IYermakov — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
-# Snakes
-wewre
+# Snakes web application
+Snakes is Java-based web application hosted on Tomcat server.
+The main goal is not to create complex functionality, but configure optimal CI/CD, git, tag flow and other DevOps related processes.
+
+ 
+## Application requirements
+Install the Java 8 JDK. The java compiler is required to run the build script. 
+Install Tomcat 8+ to run compiled WAR file.
```
Comment (mike-unitskyi) [inline README.md:8]: would add commands

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+
+## Manual deployment
+To download, build and deploy the project:
+- Clone the project
```
Comment (mike-unitskyi) [inline README.md:12]: would add commands

File: README.md [hunk 4]
```diff
@@ -1,2 +1,36 @@
...
+Open [localhost:8080](http://localhost:8080/) in a web browser to check the running application.
+
+## Automatic deployment with Jenkins
+Repository consist two Jenkins pipeline scripts for deployment management.
+Deployment works with Docker image and use AWS ECS/ECR services.
+
+##### Jenkinsfile 
+Is Managing deployment of new AWS CFN Stack. Using "Build with parameters" we can manipulate with new AWS CloudFormation stack parameters.
+
+##### Jenkinsfile2
```
Comment (mike-unitskyi) [inline README.md:29]: would rename, but it's probably out of scope of this PR

File: README.md [hunk 5]
```diff
@@ -1,2 +1,36 @@
...
+## Manual deployment
+To download, build and deploy the project:
+- Clone the project 
+- Run build.sh to compile the web app and create a WAR file
+
+**IMPORTANT**
+Always run build.sh from the root of the project directory.
+The script compiles the project's classes, packs the necessary files into a web archive
+
+- Paste WAR file to Tomcat server.
```
Comment (mike-unitskyi) [inline README.md:19]: add commands

File: README.md (hunk 4 above)
Comment (velychkovskyi) [reply README.md:29]: We are planning to combine both of them together, but this task needs more time for testing we have before Demo presentation

File: README.md (hunk 2 above)
Comment (velychkovskyi) [reply README.md:8]: Added commands for manual installation

File: README.md (hunk 3 above)
Comment (velychkovskyi) [reply README.md:12]: Added commands for manual git clone

File: README.md (hunk 5 above)
Comment (velychkovskyi) [reply README.md:19]: Added commands for manual delivery

**Pull request merged**